src/sonic-config-engine/sonic-cfggen

```diff
@@ -368,6 +364,7 @@ def main():
         if args.namespace is None:
             configdb = ConfigDBPipeConnector(use_unix_socket_path=use_unix_sock, **db_kwargs)
         else:
+            SonicDBConfig.load_sonic_global_db_config(namespace=args.namespace)
```

Comment:
Can you elaborate how this change help fix the problem ?

Reply:
We are facing issue when first time sonic installed and booted. databaseX dockers doesnt mount platform directory because     PLATFORM=${PLATFORM:-`$SONIC_CFGGEN -H -v DEVICE_METADATA.localhost.platform`} in /usr/bin/database.sh exit with exception and hence platform string is empty. 

To overcome this issue, I moved the loading database config only for reading or writing from db. Other use cases like reading PLATFROM string (first case), it doesnt require database_config.json.